include/msg.h

```diff
@@ -35,7 +35,9 @@ enum {
 	/* Directory operations */
 	mtLookup, mtLink, mtUnlink, mtReaddir,
 
-	mtCount
+	mtCount,
+
+	mtStat = 0xf53
```

Comment:
Maybe add a comment that this was moved from libphoenix and that's the reason for such a specific value?